src/containers/BuyManaWithFiatModal/BuyManaWithFiatFeedbackModal/BuyManaWithFiatFeedbackModal.tsx

```diff
@@ -5,20 +5,30 @@ import {
 } from 'decentraland-ui/dist/components/BuyManaWithFiatModal/FeedbackModal'
 import { t } from '../../../modules/translation/utils'
 import { Props } from './BuyManaWithFiatFeedbackModal.types'
+import { Network } from '@dcl/schemas'
```

Comment:
can we move this one a few lines above? 😄

Reply:
🙏